Makefile

```diff
@@ -7,17 +7,21 @@ SENTRY_PROJECT=ng-demo
 VERSION=`sentry-cli releases propose-version`
 PREFIX=dist
 
+
 setup_release: create_release associate_commits upload_sourcemaps
 
+
+
 create_release:
 	sentry-cli releases -o $(SENTRY_ORG) new -p $(SENTRY_PROJECT) $(VERSION)
+	
 
 associate_commits:
-	sentry-cli releases -o $(SENTRY_ORG) -p $(SENTRY_PROJECT) set-commits --auto $(VERSION)
+	-sentry-cli releases -o $(SENTRY_ORG) -p $(SENTRY_PROJECT) set-commits --local $(VERSION) 
 
 upload_sourcemaps:
 	sentry-cli releases -o $(SENTRY_ORG) -p $(SENTRY_PROJECT) files \
 		$(VERSION) upload-sourcemaps --url-prefix "~/" --rewrite --validate $(PREFIX)
 
-reference_release:
-	sed -i -e "s/release: .*/\release: \"${VERSION}\"/g" src/app/app.module.ts
+create_env:
```

Comment:
can you notate in the readme how this command is intended to be used?

Does it belong under Setup (i.e. command you should only have to run once) or Run (need to run it every time you want to run this angular app)

Reply:
@thinkocapo create_env is used as part of predeploy, you never run it directly, when you run deploy, predeploy kicks in, I will note this in the readme

Reply:
ok @sentryadam0000345 thanks that makes sense.